about.html

```diff
@@ -0,0 +1,166 @@
+    />
+    <link
+      rel="stylesheet"
+      href="./about/aboutMobile.css"
+      media="screen and (max-width: 768px)"
+    />
+    <link
+      rel="stylesheet"
+      href="./about/aboutDesc.css"
+      media="screen and (min-width: 768px)"
+    />
+    <title>About/page/hyundai</title>
+  </head>
+  <body>
+    <header>
+      <nav class="h_mobile mobile-menu">
+        <i class="fas fa-bars menu-bars"></i>
+        <div class="dropMenu">
+          <i class="fas fa-times close-icon"></i>
+          <ul class="menu-list">
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#courses">Contact</a></li>
+            <li class="login">
+              <a href="#"><span>BUY YOUR </span>CAR NOW</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <div class="top_icons">
+        <ul>
+          <li>
+            <a><i class="fab fa-facebook-f"></i></a>
+          </li>
+          <li>
+            <a><i class="fab fa-twitter"></i> </a>
+          </li>
+          <li>Languages</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+      <nav class="top_nav">
+        <a
+          href="https://www.freepnglogos.com/pics/hyundai-logo"
+          title="Image from freepnglogos.com"
+          ><img
+            src="https://www.freepnglogos.com/uploads/hyundai-logo-0.png"
+            alt="hyundai logo emblem png"
+        /></a>
+        <ul>
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Job Offer</a></li>
+          <li><a href="#">Help</a></li>
+          <li class="login">
+            <a href="#"><span>BUY YOUR </span>CAR NOW</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="paragraphe">
+        <div class="paragraphe_title">
+          <h1>Hyundai Rwanda. Sales and Marketting Place</h1>
+
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo
+            doloremque tempora voluptas maxime? Nulla ex, magni odit nihil
+            aspernatur itaque fugiat numquam odio veritatis placeat perferendis,
+            enim suscipit quas ipsa blanditiis! Et consequatur reiciendis
+            temporibus, animi earum ipsa id distinctio dolor tempora mollitia,
+            similique?
+          </p>
+          <small>
+            If you want a car, please contact using this email
+            www.hyundai.rw</small
+          >
+        </div>
+      </section>
+
+      <section id="courses" class="my-team">
+        <p>VISIT OUR BRAND ALL OVER THE WORLD</p>
+        <hr />
+        <div class="teamList">
+          <div class="listPara">
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod,
+              corrupti doloribus. Quam temporibus cum culpa fugiat. Quae aut
```

Comment:
- [ ] Kindly use the `background-color` recommended for this section. I believe the background color used is `#fff`. Kindly implement it 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/153725425-da6924e1-7737-45ce-93a4-5ac4c285746e.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/153725458-6be81cee-603e-4437-90d0-2c118f415374.png)